src/java/fr/paris/lutece/portal/service/dashboard/DashboardService.java

```diff
@@ -190,7 +190,7 @@ public void doMoveDashboard( IDashboardComponent dashboard, int nOldColumn, int
         {
             if ( AppLogService.isDebugEnabled( ) )
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.